packages/components/drag-handle/src/DragHandle.styles.ts

```diff
@@ -1,7 +1,7 @@
-import { css } from 'emotion';
+import { cx, css } from 'emotion';
 import tokens from '@contentful/f36-tokens';
 
-export const styles = {
+export const getStyles = () => ({
```

Comment:
doesn't this mean on each render emotion will generate the same styles with different classnames?

Reply:
no, it won't because class name value is a hash of properties and their values, new class name and styles will be generated only if values will be different.